tests/contrib/logging/test_tracer_logging.py

```diff
@@ -231,23 +232,9 @@ def test_warn_logs_can_go_to_file(run_python_code_in_subprocess, ddtrace_run_pyt
     log_file = tmpdir.strpath + "/testlog.log"
     env["DD_TRACE_LOG_FILE"] = log_file
     env["DD_TRACE_LOG_FILE_SIZE_BYTES"] = "200000"
-    patch_code = """
+    code = """
 import logging
 import ddtrace
-
-ddtrace_logger = logging.getLogger('ddtrace')
-assert ddtrace_logger.getEffectiveLevel() == logging.WARN
-assert len(ddtrace_logger.handlers) == 1
-assert isinstance(ddtrace_logger.handlers[0], logging.handlers.RotatingFileHandler)
-assert ddtrace_logger.handlers[0].maxBytes == 200000
-assert ddtrace_logger.handlers[0].backupCount == 1
-
-ddtrace_logger.warning('warning log')
-"""
-
-    ddtrace_run_code = """
-import logging
-
```

Comment:
I'm not sure if we can actually merge the `patch_code` and `ddtrace_run_code` into one code block. I thought the point of this test is to test the different execution methods of code (import ddtrace, or ddtrace-run) and make sure they're the same, but by importing ddtrace in the `ddtrace-run` version of the code doesn't that defeat the purpose?